src/middlewares/childLogger.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const assert = require('assert');
+
+module.exports = setup;
+
+// // //
+
+/**
+ * Append a logger to the request. The logger is created with `requestId: req.requestId`
+ * @see middleware requestId.js
+ * @param  {Object} logger The logger (must have a .child method)
+ * @return {Function}        the middleware
+ */
+function setup(logger) {
+  assert.ok(logger, 'Logger is missing');
```

Comment:
you can also assert here that it respects the contract:  (must have a .child method)